mpmp/src/cmds/Chat.java

```diff
@@ -19,17 +19,30 @@ public void exec(String line, Conn conn) {
 			c.sendErr(ErrCode.NotSubscribed);
 			return;
 		}
-
-		argpos = line.indexOf(' ');
-		if(argpos < 0) {
-			argpos = line.length();
-		} else {
-			while(argpos < line.length() && Character.isWhitespace(line.codePointAt(argpos)))
-				argpos++;
+		
+		String[] args = line.split("@");
+		if (args.length > 2) {
+			c.sendErr("Don't put a '@' in your message please.");
```

Comment:
This is stupid. A single `@` doesn't harm anybody. If we can't find a user after the sign, it should be interpreted as an ordinary character.